templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -11,9 +11,10 @@ contents:
       dst_path="/etc/NetworkManager/system-connections"
       if [ -d $src_path ]; then
         echo "$src_path exists"
-        fileList=$(echo {br-ex,ovs-if-br-ex,ovs-port-br-ex,ovs-if-phys0,ovs-port-phys0}.nmconnection)
+        # NetworkManager 1.15.2 or later now requires that files in /{etc,run}/NetworkManager/system-connections end in the suffix '.nmconnection'
```

Comment:
I'm not sure this even works. We had NM bugs like:
https://bugzilla.redhat.com/show_bug.cgi?id=1852106

which require minimum NM package versions in rhel 7.9z. See:
https://bugzilla.redhat.com/show_bug.cgi?id=1882667#c13

Reply:
Here's the fix in NM https://github.com/NetworkManager/NetworkManager/commit/d37ad15f12bafd91cf724cda50aea7093e04bf7a. It has been merged in NM 1.18.8.

However, in my test with RHEL7.9 and NM 1.18.8-2.el7_9, the NM still generates the keyfile names without the extension 'nmconnection'.

```
[root@rhel-worker ~]# ls /etc/NetworkManager/system-connections-merged/
br-ex  ovs-if-br-ex  ovs-if-phys0  ovs-port-br-ex  ovs-port-phys0
```